force-app/main/default/lwc/nksSakOgDokumentListeNarrow/nksSakOgDokumentListeNarrow.js

```diff
@@ -51,25 +53,44 @@ export default class NksSakOgDokumentListeNarrow extends LightningElement {
     _selectedThemeGroups = '';
     isLoaded = false;
     personId;
+    journalPostThemeSet = new Set();
 
     set brukerId(value) {
         this.queryVariables.brukerId.id = value;
         this.callGetJournalPosts(false);
         this.callGetCases();
     }
 
+    get totaltNumOfJournalPosts() {
+        if (this.sideInfo) return this.sideInfo.totaltAntall;
```

Comment:
Vi kan evt. lage en oneliner hvor vi returnerer totalAntall eller 0 dersom sideinfo mangler.
```suggestion
        return this.sideInfo ? this.sideInfo.totaltAntall : 0;
```